onchain/rollups/contracts/library/LibOutputValidation.sol

```diff
@@ -44,6 +39,22 @@ struct OutputValidityProof {
 library LibOutputValidation {
     using CanonicalMachine for CanonicalMachine.Log2Size;
 
+    /// @notice Raised when some `OutputvalidityProof` variables does not match
```

Comment:
`OutputvalidityProof` ➡️ `OutputValidityProof`

This applies to several comments added by this PR.